ui/modal/modalPublishPreview/view.jsx

```diff
@@ -160,6 +163,8 @@ const ModalPublishPreview = (props: Props) => {
     }
   } else if (livestream) {
     modalTitle = releasesInFuture ? __('Schedule Livestream') : __('Create Livestream');
+  } else if (fileText) {
+    modalTitle = __('Confirm Post');
```

Comment:
Steps:
1. Try to create a post, and then cancel when publish-preview modal appears.
2. Try to upload a video ... the publish-preview title stays at 'Confirm Post'.

If it involves extra states or more logic to clear things, maybe can just make Post and Upload use a neutral string like "Confirm Publish".  Just an idea.

<img width="352" alt="image" src="https://user-images.githubusercontent.com/64950861/177073425-432c1188-fa0b-4306-ba5e-af315097a829.png">

Reply:
Fixed.